### ISSUE TITLE
How to change shell (zsh/bash): difference with iTerm2 (Q with A)

### ISSUE THREAD

**Issue by marcelloromani** [alacritty/alacritty#6377]
I am reporting this because it's the kind of information I would have liked to find when trying to answer the question.

### System

OS macOS
Version: `alacritty 0.10.1 (2844606)`

### Question

Coming from iTerm2, I was expecting a way to change the default shell launched when an Alacritty window is created.
Alacritty doesn't have a fancy UI for options, so I found you can use `~/.alacritty.yaml`. Still no luck.

Then I realised MacOS has a _standard OS command to change the shell_ for the user.

Which is even printed when a terminal is opened.

```$ chsh -s /bin/bash```

job done.

TL;DR: Alacritty does a good job (IMO) not duplicating basic configuration functionality that is already available in the OS.

One possible downside of this is people wanting to use different shells in different windows. That's not my use case, so I am biased here.

**Comment by marcelloromani** [comment]
I posted this to leave a searchable tray, so closing as self-answered question (in StackOverflow fashion).
